dev/TeachingTip/TestUI/TeachingTipPage.xaml.cs

```diff
@@ -85,23 +85,6 @@ private void TeachingTipInVisualTree_Closed(TeachingTip sender, TeachingTipClose
 
         protected override void OnNavigatedFrom(NavigationEventArgs e)
         {
-            if (this.TeachingTipInResources != null && this.TeachingTipInResources.IsOpen)
-            {
-                this.TeachingTipInResources.IsOpen = false;
-            }
-            if (this.TeachingTipInVisualTree != null && this.TeachingTipInVisualTree.IsOpen)
-            {
-                this.TeachingTipInVisualTree.IsOpen = false;
-            }
-            if (testWindowBounds != null && testWindowBounds.IsOpen)
```

Comment:
Sorry, the testWindowBounds should still be closed... Add back in

```
            if (testWindowBounds != null && testWindowBounds.IsOpen)	
            {	
                testWindowBounds.IsOpen = false;	
            }	
            if(testScreenBounds != null && testScreenBounds.IsOpen)	
            {	
                testScreenBounds.IsOpen = false;	
            }
```

Reply:
Fixed with latest commit.